src/jit/emitxarch.cpp

```diff
@@ -1925,7 +1925,7 @@ UNATIVE_OFFSET      emitter::emitInsSizeAM(instrDesc* id, size_t code)
         // Most 16-bit operands will require a size prefix .
         // This refers to 66h size prefix override.
 
-        if  (    (attrSize == EA_2BYTE)
+        if  (    attrSize == EA_2BYTE
```

Comment:
Actually, did you mean enclosing the whole if statement or just the condition? In other words, did you mean this:
```C
        if  (
#if FEATURE_STACK_FP_X87
                 (attrSize == EA_2BYTE)
              && (ins != INS_fldcw)
              && (ins != INS_fnstcw)
#else // FEATURE_STACK_FP_X87
                attrSize == EA_2BYTE
#endif // FEATURE_STACK_FP_X87
              )
        {
            size++;
        }
```
or this?
```C
#if FEATURE_STACK_FP_X87
        if  (
                 (attrSize == EA_2BYTE)
              && (ins != INS_fldcw)
              && (ins != INS_fnstcw)
              )
#else // FEATURE_STACK_FP_X87
        if  (attrSize == EA_2BYTE)
#endif // FEATURE_STACK_FP_X87
        {
            size++;
        }
```